met-api/src/met_api/constants/subscription_type.py

```diff
@@ -0,0 +1,23 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Constants for subscription."""
+from enum import IntEnum
+
+
+class SubscriptionType(IntEnum):
+    """Enum of Subscription Type."""
+
+    ENGAGEMENT = 1
+    PROJECT = 2
+    TENANT = 3
```

Comment:
please write some comments about what does it do..
like what is subscription type and what each type does..

Reply:
Added the comments.